qa/tasks/cephadm.py

```diff
@@ -155,20 +155,26 @@ def download_cephadm(ctx, config, ref):
             )
 
 @contextlib.contextmanager
-def ceph_log(ctx, config):
-    cluster_name = config['cluster']
-    fsid = ctx.ceph[cluster_name].fsid
-
-    # Add logs directory to job's info log file
-    with open(os.path.join(ctx.archive, 'info.yaml'), 'r+') as info_file:
+def update_archive_setting(archive_dir, key, value):
```

Comment:
why is this method decorated using `contextlib.contextmanager`?

Reply:
Also `update_archive_setting` seems to be a strange name, as this function actually **doesn't update the archive setting**.

Can you find a better name?

Reply:
@tchaikov made updates, thanks for feedback!
@sebastian-philipp what about `s/update_archive_setting/update_info_yaml` ?